### DISCUSSION TITLE
Renovate Missing Schedule

### DISCUSSION THREAD

**Discussion by shenke-adsk** [mend/renovate-ce-ee#451] [Q&A]
We have a repo that constantly misses Renovate schedule. Our schedule for this repo is below:
`"extends": ["config:base", "schedule:weekends", ":maintainLockFilesWeekly", ":disableRateLimiting", ":automergeAll"],`
However, the repo does not get processed over the weekend. It runs on Monday. Any possible cause? We currently just run Renovate in a single ECS container. If we increase the task count - have the same Renovate CE image run in multiple tasks in parallel, will that help?

**Comment by rarkins** [comment]
Most probable cause is that your CE instance is overloaded and only runs each repo every X days.

CE is not designed for horizontal scalability and scaling multiple can lead to conflicts - you need to check out EE
